index.html

```diff
@@ -26,15 +26,133 @@ <h2 class="introh2">LET’S CONNECT</h2>
+      </ul>
+      <button class="card2-button">See project</button>
+    </section>
+</section>
+
+<section class="works3">
+  <section class="card3">
+    <img class="card3-img" src="./images/Port 3.png" alt="This is an image for work one">
+      <h1 class="card3-h">Tonic</h1>
+          <ul class="card3-ul1">
+              <li class="card3-canopy">CANOPY</li>
+              <li class="card3-back-end-dev">Back End Dev</li>
+              <li class="card3-2015">2015</li>
+          </ul>
+      <p class="card3-paragraph">A daily selection of privately <br> personalized reads; no accounts or <br> sign-ups required.</p>
+      <ul class="card3-ul2">
+          <li class="card3-html">HTML</li>
+          <li class="card3-css">CSS</li>
+          <li class="card3-javascript">JavaScript</li>
+      </ul>
+      <button class="card3-button">See project</button>
+    </section>
+</section>
+
+<section class="works4">
+  <section class="card4">
+    <img class="card4-img" src="./images/port 4.png" alt="This is an image for work one">
+      <h1 class="card4-h">Multi-Post <br> Stories</h1>
+          <ul class="card4-ul1">
+              <li class="card4-canopy">CANOPY</li>
+              <li class="card4-back-end-dev">Back End Dev</li>
+              <li class="card4-2015">2015</li>
+          </ul>
+      <p class="card4-paragraph">A daily selection of privately <br> personalized reads; no accounts or <br> sign-ups required.</p>
+      <ul class="card4-ul2">
+          <li class="card4-html">HTML</li>
+          <li class="card4-css">CSS</li>
+          <li class="card4-javascript">JavaScript</li>
+      </ul>
+      <button class="card4-button">See project</button>
+    </section>
+</section>
+
+<!-- ABOUT MY SELF -->
+
+<section class="about-myself">
+  <section class="container2">
+    <div class="section1">
+      <h1 class="about-myself-intro">About Myself</h1>
+
+      <p class="about-my-self-intro-p">Hello I’m a software developer! I can help you build a product , feature or website. Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.</p>
+
+      <h2 class="about-my-self-introh2">LET’S CONNECT</h2>
+
+      <div class="socials">
+        <ul>
+          <li class="ls"><a href="#" class="twitter-hp"><img src="./images/twitter.svg" alt="This is a twitter button"></a></li>
+          <li class="ls"><a href="#" class="linkedin-hp"><img src="./images/Linkedin icon.svg" alt="Thi is a linkedin button"></a></li>
+          <li class="ls"><a href="#" class="unknw1"><img src="./images/Vector.svg" alt=""></a></li>
+          <li class="ls"><a href="#" class="github-hp"><img src="./images/github.svg" alt="This is a github button"></a></li>
+          <li class="ls"><a href="#" class="unknw2"><img src="./images/Vector (1).svg" alt=""></a></li>
+        </ul>
+        <button class="get-my-resume-button">Get my resume</button>
+      </div>
```

Comment:
For this section of the website, I recommend the following:


- To make the code well structured, consider not to wrap a section tag within another since it can cause some problems with the accessibility of your website and can also make it hard to follow by other devs
- The heading tag for the project card should not be an h1 tag since this tag is mostly used to open up a website mostly at the hero section, hence I recommend using maybe an h3 here instead of h1 to follow best practices.
- Since the languages, skills and frameworks are a list, I think it'd be better to arrange the language part inside the `ul` in order to be semantically okay.
- The section should be aligned with grid and not flex here in order to be on same page with the requirements.